src/Turing.jl

```diff
@@ -66,6 +66,8 @@ end
 ###########
 # Exports #
 ###########
+# `using` statements for stuff to re-export
+using DynamicPPL: elementwise_loglikelihoods, generated_quantities
```

Comment:
How's this @devmotion ?

Reply:
Good - if we don't want to reexport everything :slightly_smiling_face:

Reply:
Hmm, maybe. Only reason not to IMO is that a lot of the functionality in DynamicPPL is meant for developers rather than end-users (people who run models, rather than developing samplers). And I think for end-users it's nice to just have the "outwards-facing" functionality, and not clutter the namespace with `VarInfo`, etc.

BUT maybe we  also should at least re-export `logpior` and siblings?

Reply:
Any other methods?

Reply:
Resxporting `logprior` and co. sounds like a good idea to me.

Reply:
Sounds reasonable, so maybe just add `logjoint` and `logprior` as well for now (`loglikelihood` is already exported since Distributions is reexported).

Reply:
Done diddit!